source/opcode.h

```diff
@@ -68,6 +68,10 @@ int32_t spvOpcodeIsConstant(const SpvOp opcode);
 // Returns true if the given opcode is a constant or undef.
 bool spvOpcodeIsConstantOrUndef(const SpvOp opcode);
 
+// Determines if the given opcode is a scalar specialization constant. Returns
+// zero if false, non-zero otherwise.
+int32_t spvOpcodeIsScalarSpecConstant(const SpvOp opcode);
```

Comment:
Is there a reason you haven't used `bool` as the return type? A boolean return type sounds more intuitive based on what the function does.